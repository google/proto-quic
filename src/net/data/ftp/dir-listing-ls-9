total 510528
-rw-r--r--  1 nobody  nogroup  174680068 Jun  4 23:20 Akademia Teatralna spot.mpg
-rw-r--r--  1 nobody  nogroup    3447432 May 18  2009 Foo - Instrukcja_Obs≈Çugi.pdf
-rw-r--r--  1 nobody  nogroup   23197684 Jun  9 13:36 Zdjecia.zip
