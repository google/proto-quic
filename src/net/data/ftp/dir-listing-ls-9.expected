-
Akademia Teatralna spot.mpg
174680068 
1994
6
4
23
20

-
Foo - Instrukcja_Obs≈Çugi.pdf
3447432 
2009
5
18
0
0

-
Zdjecia.zip
23197684 
1994
6
9
13
36
